assets/css/style.css

```diff
@@ -163,15 +162,370 @@ button:hover {
   transform: translate(2px, 2px);
 }
 
+.work-section {
+  margin-top: 140px;
+  width: 375px;
+}
+
+.second-title {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  color: #1c1a19;
+  width: 327px;
+  height: 83px;
+  margin: 0 auto 4%;
+  text-align: center;
+}
+
+.line_below_title {
+  width: 327px;
+  height: 0;
+  border: 0.4px solid #212121;
+  margin: 0 auto 8%;
+  margin-bottom: 100px;
+}
+
+.card-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 375px;
+  margin-bottom: 60px;
+}
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓